packages/web-components/src/components/va-crisis-line-modal/va-crisis-line-modal.tsx

```diff
@@ -48,6 +48,7 @@ export class VACrisisLineModal {
           </button>
         </div>
         <va-modal
```

Comment:
For some reason you cannot navigate the modal with the keyboard. I was testing with voice over on and was trying to navigate by using ctrl option space and also by just using the tab key. I could never get the focus to move from the close button.